src/main/java/at/hannibal2/skyhanni/features/chat/playerchat/PlayerChatModifier.kt

```diff
@@ -80,7 +80,7 @@ class PlayerChatModifier {
 
         if (SkyHanniMod.feature.markedPlayers.highlightInChat) {
             for (markedPlayer in MarkedPlayerManager.playerNamesToMark) {
-                string = string.replace(markedPlayer, "§e$markedPlayer§r")
+                string = string.replace(markedPlayer, "§e$markedPlayer§r", true)
```

Comment:
please remove this from this pr and move it into a standalone pr

Reply:
oops. idk why thats there